eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/EventMeshTcpMessageDispatcher.java

```diff
@@ -77,8 +76,45 @@ protected void channelRead0(ChannelHandlerContext ctx, Package pkg) throws Excep
             dispatch(ctx, pkg, startTime, cmd);
         } catch (Exception e) {
             logger.error("exception occurred while pkg|cmd={}|pkg={}|errMsg={}", cmd, pkg, e);
```

Comment:
The `errMsg={}` in the error log is redundant. It is better midify to 
```java
logger.error("exception occurred while pkg|cmd={}|pkg={}", cmd, pkg, e);
```

Reply:
Thank you, I got it, and I will fix it later.